_rules/html-page-non-empty-title-2779a5.md

```diff
@@ -198,6 +198,27 @@ This page has a `title` element that only contains a separator character.
 </html>
 ```
 
+#### Failed Example 6
+
+This page does not have a title because the shadow root is not a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) of the [document element](https://dom.spec.whatwg.org/#document-element).
+
+```html
+<html>
+	<body>
+		<template id="shadow-element">
+			<title>This is the page title</title>
+		</template>
+		<script>
+      			const host = document.querySelector("body");
+      			const shadow = host.attachShadow({ mode: "closed" });
```

Comment:
Nitpick; I think this is a slightly stronger test case if it uses open shadow DOM, since closed shadow DOM can't be tested with regular DOM APIs.

```suggestion
      			const shadow = host.attachShadow({ mode: "open" });
```